.github/workflows/changelog-v3.yml

```
@@ -1,25 +1,191 @@
 jobs:
   validate:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      actions: write
+    
     steps:
-    - uses: actions/checkout@v4
-    - uses: actions/setup-go@v4
+    - name: Checkout PR code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)